open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerAdminStoreServices.java

```diff
@@ -215,6 +242,58 @@ private OMAGServerConfigStore getServerConfigStore(String   serverName,
                                                     error);
         }
     }
+    /**
+     * Retrieve the connection to the config files.
+     *
+     * @param methodName  method requesting the server details
+     * @return configuration connector file
+     * @throws OMAGConfigurationErrorException A connection error occurred while attempting to access the server config store.
+     */
+    private OMAGServerConfigStoreRetrieveAll getServerConfigStoreForRetrieveAll(
+                                                       String   methodName) throws OMAGConfigurationErrorException
+    {
+        Connection   connection = this.getConnectionForRetrieveAll();
+
+        try
+        {
+            ConnectorBroker connectorBroker = new ConnectorBroker();
+
+            Connector connector = connectorBroker.getConnector(connection);
+
+            OMAGServerConfigStore serverConfigStore = (OMAGServerConfigStore) connector;
```

Comment:
Seems that the retrieve all connector should be a different type of connector to the existing configuration store